package.json

```diff
@@ -247,6 +254,7 @@
     "statsig-react": "^1.22.0",
     "styled-components": "^5.3.5",
     "tiny-invariant": "^1.2.0",
+    "ts-jest": "^29.1.1",
```

Comment:
Do we need to use ts-jest, or can we use babel-jest to avoid adding a new dependency?